deluge/ui/gtkui/glade/add_torrent_dialog.ui

```diff
@@ -313,8 +361,6 @@
                     <property name="visible">True</property>
                     <property name="can_focus">True</property>
                     <property name="border_width">5</property>
-                    <property name="hscrollbar_policy">automatic</property>
-                    <property name="vscrollbar_policy">automatic</property>
                     <property name="shadow_type">out</property>
                     <child>
                       <object class="GtkTreeView" id="listview_files">
```

Comment:
To make it clearer what is being references in code, should rename `listview_files` to `treeview_files` and obviously change all references in  `deluge/ui/gtkui/addtorrentdialog.py`